Plugins/PluginUtilities/PluginUtilities.cpp

```diff
@@ -938,6 +938,100 @@ Quaternion HkMatrixToQuaternion(Matrix m)
 	return quaternion;
 }
 
+Matrix TransposeMatrix(Matrix& m)
+{
+	return { {
+		{m.data[0][0], m.data[1][0], m.data[2][0]},
+		{m.data[0][1], m.data[1][1], m.data[2][1]},
+		{m.data[0][2], m.data[1][2], m.data[2][2]}} };
+}
+
+float MatrixDeterminant(Matrix& m, uint row, uint col)
+{
+	float minorMatrix[4];
+	uint counter = 0;
+	for (int i = 0; i < 3; i++)
+	{
+		if (i == row)
+			continue;
+		for (int j = 0; j < 3; j++)
+		{
+			if (j == col)
+				continue;
+			minorMatrix[counter] = m.data[i][j];
+			counter++;
+		}
+	}
+	return minorMatrix[0] * minorMatrix[3] - minorMatrix[1] * minorMatrix[2];
```

Comment:
Shouldn't either this function, or the callers, be multiplying the result by m[row][col]?